ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -628,7 +634,7 @@ private class DisplayURLLabel: UILabel {
     super.layoutSubviews()
 
     clippingFade.frame = .init(
-      x: isRightToLeft ? bounds.width - 20 : 0,
+      x: isRightToLeft && isWebScheme ? bounds.width - 20 : 0,
```

Comment:
This should be `isRightToLeft || !isWebScheme` since you want the clipping on the trailing edge if its not a web scheme

Reply:
Fixed everything: https://github.com/brave/internal/issues/1210#issuecomment-2427950790
and verified everything is working properly again.

So Arabic is displaying the URLs correctly.